src/components/dialog/ai-dialog.less

```
@@ -0,0 +1,102 @@
+@import '../../assets/less/settings.less';
+
+@animationSpeed: .2s;
+
+.ai-dialog {
+  position: relative;
+  font-size: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .wrapper {
+    position: absolute;
+    padding: 0.5em;
+    background-color: white;
+
+    .content-wrapper {
+      position: absolute;
+      background-color: white;
+      opacity: 0;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      pointer-events: none;
+      bottom: 0;
+      padding-bottom: ~'calc(0.5em + 1px)';
+      transition: width @animationSpeed ease-out, height @animationSpeed ease-out, padding ease-out @animationSpeed;
+      left: 50%;
+      transform: translateX(-50%);
+      overflow: hidden;
+
+      .content {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        width: 100%;
+        height: 100%;
+        color: black;
+        border: 1px solid black;
+        transition: @animationSpeed;
+        box-sizing: border-box;
+        span {
+          flex: 0 0 auto;
+          position: relative;
+          font-size: 1.5rem;
+          padding: 0 1em;
+          transition: @animationSpeed;
+          background-color: white;
+          top: -.8em;
+        }
+        .slot-wrapper {
+          display: flex;
+          justify-content: center;
+          height: 0em;
+          opacity: 0;
+          flex: 1 1 auto;
+          width: 100%;
+          height: 100%;
+          transition: @animationSpeed;
+          transform: translateY(-2em);
+        }
+      }
+    }
+  }
+  &.open .wrapper .content-wrapper {
+    width: 25em;
+    height: 32em;
+    padding: 2em;
+    bottom: 0;
+    pointer-events: all;
+    .content {
+      border: 2px solid black;
+    }
+  }
+}
+
+button {
+  color: @fontColor;
+  font-family: 'Source Sans Pro';
+  font-size: 1.5rem;
+  padding: .5em 1.5em;
+  box-sizing: border-box;
+  border: 1px solid black;
+  background-color: white;
+  white-space: nowrap;
+  transition: 0@animationSpeed;
```

typo? transition: **0**@animationSpeed;
```suggestion
transition: @animationSpeed;
```